packages/devtools_app/lib/src/screens/debugger/breakpoint_manager.dart

```diff
@@ -93,6 +93,10 @@ class BreakpointManager with DisposerMixin {
     if ([
       EventKind.kPauseStart,
       EventKind.kPausePostRequest,
+      // We check for a resume event because package:dwds sends a resume event
+      // after a hot-restart. See:
+      // https://github.com/flutter/devtools/issues/9124
+      EventKind.kResume,
```

Comment:
We should make sure this doesn't cause issues for non-web apps.

Reply:
Agreed! Testing now

Reply:
Confirmed non-web apps still work as expected. Also looking into what is required to reenable the debugger panel integration test on web, since I believe that would have caught this regression: https://github.com/flutter/devtools/issues/7732